src/modules/plotter.py

```diff
@@ -0,0 +1,134 @@
+import json
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+
+class HarryPlotter:
```

Comment:
Plots should be saved under their model and mode subdirectories: 'src/plots/mappo/training/mappo_2_0.0_training_joint_action_frequencies_plot.png'